packages/icons/src/index.js

```diff
@@ -81,6 +81,7 @@ export { default as formatUnderline } from './library/format-underline';
 export { default as formatUppercase } from './library/format-uppercase';
 export { default as fullscreen } from './library/fullscreen';
 export { default as gallery } from './library/gallery';
+export { default as globalStyles } from './library/global-styles';
```

Comment:
Wonder if we should be calling this one just `styles`

Reply:
I've no strong opinion. I case we want to do it, [here's a PR](https://github.com/WordPress/gutenberg/pull/34946). The argument for calling it "global styles" was that it feels like a branded symbol that should not be used for any other interface.